resources/views/pages/home.blade.php

```diff
@@ -1,8 +1,188 @@
 <x-general-layout>
-    <div class="container mx-auto py-5 prose mb-32">
-        <span class="text-eph-purple font-bold">Online fysiotherapie</span>
-        <h1 class="heading-h1">Waar en wanneer het jou uitkomt</h1>
-        <p class="text-eph-grey">Behandeld worden door professionele fysiotherapeuten door heel Nederland.</p>
-        <x-links.button class="btn-primary rounded-full mt-4" href="#">Hoe werkt het?</x-link-button>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
```

Comment:
Als we geen Bootstrap icoontjes gebruiken mag deze eruit toch? En anders mag je 'm in [`styles.blade.php`](resources/views/includes/styles.blade.php) zetten.